package/kedro_viz/models/flowchart.py

```diff
@@ -619,58 +590,95 @@ def __post_init__(self, data_node: DataNode):
                 )
 
 
-@dataclass
+class TranscodedDataNode(GraphNode):
```

Comment:
Why is this moved down?

Reply:
I was trying to reorder node classes with metadata node class. I reverted the re-order as it is making PR reviews difficult